Library/Homebrew/download_strategy.rb

```diff
@@ -360,7 +360,7 @@ def parse_basename(url, search_query: true)
       end
 
       if search_query && (uri_query = uri.query.presence)
-        components[:query] = URI.decode_www_form(uri_query).map(&:second)
+        components[:query] = URI.decode_www_form(uri_query).map { _1.fetch(1) }
```

Comment:
```suggestion
        components[:query] = URI.decode_www_form(uri_query).map { |_, value| value }
```

maybe

Reply:
These simplify to `_2`, which seems semantically equivalent to `second`, but i'm open to feedback that it's a bit too arcane.